### PR TITLE
Update programs-and-data.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Add peanut butter and jelly youtube video to the content

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| programs-and-data | Link checking took too long. Verify it manually. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| programs-and-data | Link checking took too long. Verify it manually. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: coding-intro/coding-intro-core/core-concepts/programs-and-data.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ type: normal
 
 category: must-know
 
+links:
+  - >-
+    [PB & J Sandwitch](youtube.com/watch?v=Ct-lOOUqmyY){website}
```
Comment (nem035) [inline coding-intro/coding-intro-core/core-concepts/programs-and-data.md:10]:
```suggestion
    [PB & J Sandwitch](https://youtube.com/watch?v=Ct-lOOUqmyY){website}
```

File: coding-intro/coding-intro-core/core-concepts/programs-and-data.md [hunk 2]
```diff
@@ -18,6 +22,8 @@ Let's quickly introduce the building blocks we'll cover in this course.
 
 Even though they're super fast, computers are annoyingly literal - you have to spell out everything for them. 
 
+> Check out the *Learn more* section to see two kids writing instructions to their programmer dad on how to make a peanut butter and jelly sandwich.
```
Comment (nem035) [inline coding-intro/coding-intro-core/core-concepts/programs-and-data.md:25]:
```suggestion
> Check out the *Learn more* section for a realistic (and funny) example of this 🤓
```

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/coding-intro/coding-intro-core/core-concepts/programs-and-data.md](https://app.enkipro.com/public/preview-insight/enkidevs/curriculum/coding-intro/coding-intro-core/core-concepts/programs-and-data.md?ref=cc92168bb5b50f9fff8e4b914ee2d2d9337833c9)

File: coding-intro/coding-intro-core/core-concepts/programs-and-data.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ category: must-know
 
 links:
   - >-
-    [PB & J Sandwitch](youtube.com/watch?v=Ct-lOOUqmyY){website}
+    [EXACT instructions Challenge](https://youtube.com/watch?v=Ct-lOOUqmyY){website}
```
Comment (nem035) [inline coding-intro/coding-intro-core/core-concepts/programs-and-data.md:10]:
```suggestion
    [EXACT Instructions Challenge](https://youtube.com/watch?v=Ct-lOOUqmyY){website}
```

**Review by nem035 — DISMISSED** [review]

File: coding-intro/coding-intro-core/core-concepts/programs-and-data.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ type: normal
 
 category: must-know
 
+links:
+  - >-
+    [EXACT Instructions Challenge](https://youtube.com/watch?v=Ct-lOOUqmyY){website}
```
Comment (kapnobatai137) [inline coding-intro/coding-intro-core/core-concepts/programs-and-data.md:10]: let's make the tag `video`

```suggestion
    [EXACT Instructions Challenge](https://youtube.com/watch?v=Ct-lOOUqmyY){video}
```

we can also say `'Exact'`. it looks better imo but it's up to you if you want to keep it as `'EXACT'`

```suggestion
    [Exact Instructions Challenge](https://youtube.com/watch?v=Ct-lOOUqmyY){video}
```

File: coding-intro/coding-intro-core/core-concepts/programs-and-data.md [hunk 5]
```diff
@@ -18,6 +22,8 @@ Let's quickly introduce the building blocks we'll cover in this course.
 
 Even though they're super fast, computers are annoyingly literal - you have to spell out everything for them. 
 
+> Check out the *Learn more* section for a realistic (and funny) example of this 🤓
```
Comment (kapnobatai137) [inline coding-intro/coding-intro-core/core-concepts/programs-and-data.md:25]:
```suggestion
> 💡 Check out the *Learn more* section for a realistic (and funny) example of this 🤓
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**